aspnet/single-page-application/overview/templates/emberjs-template.md

```diff
@@ -21,7 +21,7 @@ The EmberJS SPA template is designed to get you started quickly building interac
 
 "Single-page application" (SPA) is the general term for a web application that loads a single HTML page and then updates the page dynamically, instead of loading new pages. After the initial page load, the SPA talks with the server through AJAX requests.
 
-![](emberjs-template/_static/image1.png)
+![Diagram that shows two boxes labeled Client and Server. An arrow labeled AJAX goes from Client to Server. An arrow labeled H T M L and an arrow labeled J S O N go from Server to Client.](emberjs-template/_static/image1.png)
```

Comment:
I think JSON files are pronounced like "Jason," so we could use "J SON" instead here.